devolutions-gateway/src/interceptor/pcap_recording.rs

```diff
@@ -46,15 +92,27 @@ impl PacketInterceptor for PcapRecordingInterceptor {
         debug!("New packet intercepted. Packet size = {}", data.len());
 
         let server_info = self.server_info.lock().unwrap();
+        let is_from_server = source_addr.unwrap() == server_info.addr;
```

Comment:
source_addr: Could it be none and panic ?

Reply:
for now, the answer is not. But I added the check up in case if something will change.